crates/bevy_pbr/src/render_graph/pipeline/pbr.frag

```diff
@@ -45,9 +45,9 @@ struct Light {
 };
 
 layout(location = 0) in vec3 v_Position;
-layout(location = 1) in vec3 v_Normal;
+layout(location = 1) in vec3 v_WorldNormal;
 layout(location = 2) in vec2 v_Uv;
-layout(location = 3) in vec3 w_Position;
+layout(location = 3) in vec3 v_WorldPosition;
 
```

Comment:
For normal maps I need to add world-space tangents. To clear things up I've renamed the inputs and outputs with a `World` prefix.